src/main/java/org/embl/mobie/io/ome/zarr/loaders/N5OMEZarrImageLoader.java

```diff
@@ -199,6 +202,9 @@ private void initSetups() throws IOException {
             if (zarrAxes.is4DWithTimepointsAndChannels()) {
                 nC = attributes.getDimensions()[2];
             }
+            if (zarrAxes.is3DWithChannels()) {
+                nC = attributes.getDimensions()[2];
```

Comment:
I don't think this is correct, since I think we have the following axes:
```
XYZC
0123
```
So C should be 3.

Reply:
nvm, I guess this is the case `XYC` and it's correct ...